### PR TITLE
WINC-703: [config] adds controller manager resource limits

### PR DESCRIPTION
**Pull request by wgahnagl**
Adds resource limits for containers, taken from MCO's resource limits.

Testing done on a 4.14 GCP cluster with 10 windows worker node, and a  4.14 platform none cluster, with one windows worker node. 

ran make bundle

### CODE REVIEW

**Review by saifshaikh48 — COMMENTED** [review]
`make bundle` might be required to propagate these changes into the CSV

**Review by saifshaikh48 — COMMENTED** [review]
/lgtm

nit: it's generally good practice to put any commands you ran to generate changes in the commit message, something like "Ran `make bundle` for CSV changes"

File: bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -418,7 +418,10 @@ spec:
                 image: REPLACE_IMAGE
                 imagePullPolicy: IfNotPresent
                 name: manager
-                resources: {}
+                resources:
+                  requests:
+                    cpu: 20m
+                    memory: 50Mi
```
[the remark this answers is not in the document]
Comment (wgahnagl) [reply bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:424]: The MCO doesn't have a `limit` set for its resources. Should I check what other operators are using for their limits?

[the remark this answers is not in the document]
Comment (jrvaldes) [reply bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:424]: Yes, I'd say.   `requests` helps the scheduler to place the pod, and `limit` is actually what controls the resource boundaries.

**Comment by jrvaldes** [comment]
> `make bundle` might be required to propagate these changes into the CSV

@wgahnagl  Please mention in the PR description that this command ran.

**Comment by wgahnagl** [comment]
/test nutanix-e2e-operator

`cp: cannot stat '/tmp/installer/.openshift_install.log': No such file or directory `

Retesting to see if this is a flake

**Pull request merged**